Crate nimiq-primitives doesn't build with the feature coin

Crate `nimiq-primitives` doesn't build with the feature `coin`. This is the output building the crate for the feature `coin`:
```
3/8: coin
error[E0432]: unresolved import `lazy_static`
 --> primitives/src/coin.rs:8:5
  |
8 | use lazy_static::lazy_static;
  |     ^^^^^^^^^^^ use of undeclared crate or module `lazy_static`

error: cannot determine resolution for the macro `lazy_static`
   --> primitives/src/coin.rs:215:1
    |
215 | lazy_static! {
    | ^^^^^^^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

error[E0425]: cannot find value `COIN_PARSE_REGEX` in this scope
   --> primitives/src/coin.rs:230:24
    |
230 |         let captures = COIN_PARSE_REGEX.captures(s).ok_or_else(e)?;
    |                        ^^^^^^^^^^^^^^^^ not found in this scope

warning: unused `#[macro_use]` import
 --> primitives/src/lib.rs:1:1
  |
1 | #[macro_use]
  | ^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused import: `regex::Regex`
  --> primitives/src/coin.rs:10:5
   |
10 | use regex::Regex;
   |     ^^^^^^^^^^^^

Some errors have detailed explanations: E0425, E0432.
For more information about an error, try `rustc --explain E0425`.
error: could not compile `nimiq-primitives` due to 3 previous errors; 2 warnings emitted
Failed to build feature coin for package at primitives/
```

--- a/primitives/Cargo.toml
+++ b/primitives/Cargo.toml
@@ -41,7 +41,7 @@ lazy_static = "1.2"
 
 [features]
 all = ["coin", "account", "policy", "networks", "slots"]
-coin = ["hex", "thiserror", "num-traits", "regex"]
+coin = ["hex", "lazy_static", "thiserror", "num-traits", "regex"]
 account = ["hex", "thiserror"]
 policy = ["num-bigint", "num-traits", "parking_lot", "lazy_static"]
 networks = ["thiserror"]

--- a/primitives/src/lib.rs
+++ b/primitives/src/lib.rs
@@ -1,3 +1,4 @@
+#[cfg(any(feature = "account", feature = "networks", feature = "slots"))]
 #[macro_use]
 extern crate beserial_derive;
 

